### PR TITLE
Fix node in maintenance container

### LINKED ISSUE #1641: Node - permission denied
It seems that node does not like to be run as root.  I think this is one of the issues @patcon is running into in PR #1449 as well.  It seems to not affect docker on a regular windows file system but does appear on a WSL or Apple file system.  I think our maintenance container should be updated to not run regular commands as root.

### PR DESCRIPTION
**Pull request by petertgiles**
This branch changes the maintenance container build to solve the "permission denied" errors observed when running the container on linux (and linux in wsl).  It also removes the profiles usage that fails on older versions of docker compose.

Closes #1641

### CODE REVIEW

File: .github/workflows/check-lighthouse-schema.yml
Comment (petertgiles) [inline .github/workflows/check-lighthouse-schema.yml:26]: Use an override file for CI services since profiles fails on slightly older versions of docker compose: https://docs.docker.com/compose/extends/#multiple-compose-files

**Review by esizer — APPROVED** [review]
Nice! Ran from fresh WSL2 Ubuntu 20.04 with no issue. Both `setup.sh` and `refresh_all.sh`! 🎉

**Comment by petertgiles** [comment]
> Nice! Ran from fresh WSL2 Ubuntu 20.04 with no issue. Both `setup.sh` and `refresh_all.sh`! tada

Thanks Eric!  Any chance an Apple person could git it a whirl before I merge it?

**Comment by patcon** [comment]
> Any chance an Apple person could give it a whirl before I merge it?

Doing now!

Will then try running our setup.sh script on GitHub Actions and see if this has unfucked that too 🎉

**Comment by esizer** [comment]
Just tested this on my macbook and it worked fine on:

- MacOS 10.15.5
- Docker 4.6.1

**Comment by patcon** [comment]
Yesssss ok, this fixed our ability to run current scripts on GitHub Actions, which isssss interesting...
https://github.com/patcon/gc-digital-talent/runs/6130579542?check_suite_focus=true

That means that we're no longer blocked on #2217 to get Cypress tests running 🤔

**Pull request merged**
